src/libstd/keyword_docs.rs

```diff
@@ -1582,11 +1582,13 @@ mod type_keyword {}
 /// system.
 ///
 /// The `unsafe` keyword has two uses: to declare the existence of contracts the
-/// compiler can't check, and to declare that a programmer has checked that
-/// these contracts have been upheld. They are not mutually exclusive, as can be
-/// seen in `unsafe fn` (more details on this can be found below).
+/// compiler can't check (`unsafe fn` and `unsafe trait`), and to declare that a
+/// programmer has checked that these contracts have been upheld (`unsafe {}`
+/// and `unsafe impl`, but also `unsafe fn` -- see below). They are not mutually
+/// exclusive, as can be seen in `unsafe fn` (more details on this can be found
+/// below).
```

Comment:
This says 'see below' twice, I think we can only say it once.
```suggestion
/// and `unsafe impl`, but also `unsafe fn` -- see below). They are not mutually
/// exclusive, as can be seen in `unsafe fn`.
```

Reply:
Sure, I was just editing that first sentence.

Reply:
Fixed !